session1/challenge/client/src/js/components/container/App.js

```diff
@@ -27,28 +31,34 @@ const DataTile = ({ name, value, loading, active, onClick }) => {
 
 const DataList = ({ data, loading, onClick, principal}) => {
   const items = loading ? FAKE_DATA : data;
-  // TODO: Use the DateTile and return an array for every data
-
+  
   /* value={items[key]}
    * name={key}
    * onClick={onClick}
    * loading={loading}
    * active={key === principal}*/
-
-  return null
+  return Object.keys(items).map((key) =>{
```

Comment:
another way to do it is by using `Object.entries`, so you can avoid using `items[key]` to get the data

``` js
Object.entries(items).map(([key, item]) => (
     <TileErrorBoundary key={key}>
        <DataTile 
          value={item}
          name={key}
          onClick={onClick}
          loading={loading}
          active={key === principal} />
      </TileErrorBoundary>
    )
);
```

Reply:
Ok, i got it !!!.

I was always work with **"keys"** method, but **"entries"** is really better, because we get the each element directly, that really is very good.

Thank you so much for your comments